src/models/parentModel.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+
+const parentSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: [true, 'Please provide name']
+    },
+    userName: {
```

Comment:
based on the last sync, I thought we agreed that email will be the username? so no need for a separate username..

Reply:
Yes, correct @Austine105 Email would be username

Reply:
Thank you, I will correct